platform/web/js/libs/library_godot_javascript_singleton.js

```diff
@@ -127,6 +127,16 @@ const GodotJSWrapper = {
 			GodotRuntime.setHeapValue(p_exchange, id, 'i64');
 			return 24; // OBJECT
 		},
+
+		isBuffer: function (obj) {
+			switch (typeof obj) {
+			case 'object':
+				break;
+			default:
+				return false;
+			}
+			return obj instanceof ArrayBuffer || ArrayBuffer.isView(obj);
+		},
```

Comment:
I don't think we need the switch case at all

```suggestion
		isBuffer: function (obj) {
			return obj instanceof ArrayBuffer || ArrayBuffer.isView(obj);
		},
```